docs/chrome-flags-for-tools.md

```diff
@@ -94,6 +94,10 @@ These flags are being used in various tools. They also just need to be documente
 --disable-features=site-per-process # Disables OOPIF. https://www.chromium.org/Home/chromium-security/site-isolation
```

Comment:
add --enable-automation to this list?

Reply:
in fact we have the best docs for that particular flag. ;) https://github.com/GoogleChrome/chrome-launcher/blob/master/docs/chrome-flags-for-tools.md#--enable-automation

Reply:
oh haha nice, I wish GH had a ctrl+f for the whole file easily available without having to manually expand the undiff'd parts of it